CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 2.14.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.4.1166** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for dynamically updating ``customized_route_advertise`` in **aviatrix_aws_tgw_vpc_attachment**
+2. Implemented support for controller login SAML Endpoint in **aviatrix_saml_endpoint**
+3. Implemented **data_source_aviatrix_vpc**
+4. New resources to support AWS TGW inter region peering
+  - **aviatrix_aws_tgw_peering**
+  - **aviatrix_aws_tgw_peering_domain_conn**
+5. New resources
+  - **aviatrix_transit_external_device_conn**
+
+### Enhancements:
+1. Added ``peering_ha_gw_name`` in **aviatrix_gateway**, and ``ha_gw_name`` in **aviatrix_spoke_gateway** and **aviatrix_transit_gateway** as computed value to output HA gateway name
```

Comment:
Sounds redundant. Probably can change the ending to:

... as computed output values.